ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -249,27 +237,29 @@ constexpr auto deg2rad = ttnn::register_operation("ttnn::deg2rad", TO_LAMBDA_WIT
 constexpr auto rad2deg = ttnn::register_operation("ttnn::rad2deg", TO_LAMBDA_WITH_RESHAPE(ttnn::operations::unary::rad2deg));
```

Comment:
can be implemented like the ops below.  can be moved to unary_composite_op.cpp to match existing struct ExecuteUnaryCompositeOp